types/stripe/index.d.ts

```diff
@@ -819,7 +827,7 @@ declare namespace Stripe {
              * The individual’s gender (International regulations require either “male” or “female”).
              * This can be unset by updating the value to null and then saving.
              */
-            gender?: "male" | "female";
```

Comment:
Can you keep using double quotes to avoid a giant diff here?

Reply:
Please keep the PR diff limited to places that you're actually materially changing.

Reply:
@vaskevich I ran prettier using the prettierrc.json in the root. That config calls for singleQuotes. 
Doing changes to this file has proven quite painful without the assistance of a formatter.
Is there an overall plan for formatting on DT?

For this PR, I will start over with double quotes and no prettier when I find the time

Reply:
Maybe you can send out a PR with just the prettier changes that can be auto-approved? And then follow up with this change after.

Reply:
Good point @vaskevich thank you, I will do that.